### DISCUSSION TITLE
Hide WMS feature attributes

### DISCUSSION THREAD

**Discussion by MoRadwan74** [TerriaJS/terriajs#6204] [Q&A] [answered]
I want to hide some attributes from the feature info panel and downloading data. For example, hide the two fields as highlighted below.
![terria](https://user-images.githubusercontent.com/30727109/159436674-53bf599b-3cfb-4892-b65f-2da15bed41fa.png)

Is there a way to do this?

**Comment by hardreddata** [comment, marked as answer]
You can manually define the format of your table using feature info templates

https://docs.terria.io/guide/connecting-to-data/customizing-data-appearance/feature-info-template/

Though with this solution anyone will be able to click 'show raw data' and see the underlying data.

You could inspect that table element to get the html starting point. Probably something like

```
  "featureInfoTemplate":{
    "template":"<table><tbody><tr><td>latitude</td><td>{{lat}}</td></tr><tr><td>longitude</td><td>{{lon}}</td></tr></tbody></table>"
   }
```
